xc/xenops_server_xen.ml

```diff
@@ -1647,10 +1620,10 @@ module VM = struct
     try
       Opt.iter (fun info ->
           match vm.Vm.ty with
-          | Vm.HVM { Vm.qemu_stubdom = true } ->
-            if saved_state then failwith "Cannot resume with stubdom yet";
-            debug "Ignoring request for PV VNC in stubdom (would require qemu-trad)"
-          | Vm.HVM { Vm.qemu_stubdom = false } ->
+          | Vm.HVM { Vm.qemu_stubdom } ->
+            if qemu_stubdom then
```

Comment:
Would be good to drop qemu_stubdom field, but I think we can't do that because we won't be able to load old xenopsd metadata.

Reply:
Indeed, we have to keep it, unfortunately.